multicluster/controllers/multicluster/common/helper.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 Antrea Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import corev1 "k8s.io/api/core/v1"
+
+const (
+	AntreaMCServiceAnnotation = "multicluster.antrea.io/imported-service"
+
+	AntreaMCSPrefix   = "antrea-mc-"
+	ServiceKind       = "Service"
+	EndpointsKind     = "Endpoints"
+	ServiceImportKind = "ServiceImport"
+
+	SourceName      = "sourceName"
+	SourceNamespace = "sourceNamespace"
+	SourceKind      = "sourceKind"
+
+	DefaultWorkerCount = 5
+
+	ResourceExportFinalizer = "resourceexport.finalizers.antrea.io"
+)
+
+// TODO: Use NamespacedName stringer method instead of this. e.g. nsName.String()
+func NamespacedName(namespace, name string) string {
```

Comment:
namespace -> Namespace

Reply:
done

Reply:
Sorry, this one is my mistake. It should be "namespace" indeed.